meson.build

```diff
@@ -138,11 +138,20 @@ if scdoc.found()
 		'sway/sway-input.5.scd',
 		'sway/sway-ipc.7.scd',
 		'sway/sway-output.5.scd',
-		'swaybar/swaybar-protocol.7.scd',
 		'swaymsg/swaymsg.1.scd',
-		'swaynag/swaynag.1.scd',
-		'swaynag/swaynag.5.scd',
 	]
+
+	if get_option('swaynag')
+		man_files += [
+			'swaynag/swaynag.1.scd',
+			'swaynag/swaynag.5.scd',
+		]
+	endif
+
+	if get_option('swaybar')
+		man_files += [ 'swaybar/swaybar-protocol.7.scd' ]
```

Comment:
The swaybar protocol is still supported by sway even if the swaybar client is disabled, so I think it makes sense to keep building this man page unconditionally.